### PR TITLE
Better UI

### PR DESCRIPTION
**Pull request by Ohyeon5**

### CODE REVIEW

File: ui/app.py [hunk 1]
```diff
@@ -1,127 +1,123 @@
...
+    st.write(results)
+    actions = get_suggested_actions(question, results)
+    return actions
+
+
+
+def suggest_actions(actions: str):
+    # * Generating quotes and new short definitions from what we found, as a list of takeaways. (GPT3 Explain and Write)
+    st.subheader(
+        "List of takeaways which might be considered, according to current policies"
```
Comment (metinaydar) [inline ui/app.py]: "List of takeaways which might be considered"

Comment (metinaydar) [reply ui/app.py]: I forgot to do that before :)

**Review by metinaydar — APPROVED** [review]

**Review by lfhvn — APPROVED** [review]

File: ui/app.py [hunk 2]
```diff
@@ -1,127 +1,112 @@
...
-    
-def tldr():     
+    st.write(results)
+    actions = get_suggested_actions(question, results)
+    return actions
+
+def suggest_actions(actions: str):
+    # * Generating quotes and new short definitions from what we found, as a list of takeaways. (GPT3 Explain and Write)
+    st.subheader(
+        "List of takeaways which might be considered"
```
Comment (lfhvn) [inline ui/app.py:48]: suggestions, something like "List of takeaways you might consider:" or "Here are a list of takeaways I would consider:"

File: ui/components/image_generation.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import openai
+import streamlit as st
+
+def generate_image(text_input):
+    img_response = openai.Image.create(prompt=f"{text_input} a cartoon image", n=3, size="256x256")
```
Comment (lfhvn) [inline ui/components/image_generation.py:5]: do we feel cartoon is the best image style here? or would it be better to do something like "photorealistic futuristic image"

File: ui/components/text_completion.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      max_tokens=50,
+      top_p=1,
+      frequency_penalty=0,
+      presence_penalty=0,
+    )
+    return response_summary.choices[0].text
+
+def get_suggested_actions(question: str, summaries: str):
+
+    whoami3 = "I'm a climate change campaigner to help to make actions that the public can follow easily. I also consider which scenario you are in. If you tell me the potential risks and the scenario, I will let you know which actions to take."
```
Comment (lfhvn) [inline ui/components/text_completion.py:68]: I wonder if campaigner is the right word here? Wondering if something like "I'm a climate change expert and I can communicate actions that the public can easily take to lower their carbon footprint. If you tell me what kind of action you're thinking of taking, I can help you decide..." I know you've massaged this a lot, mostly wondering if the word campaigner is disproportionately biasing the outputs.

Comment (lfhvn) [inline ui/components/text_completion.py:68]: I wonder if campaigner is the right word here? Wondering if something like "I'm a climate change expert and I can communicate actions that the public can easily take to lower their carbon footprint. If you tell me what kind of action you're thinking of taking, I can help you decide..." I know you've massaged this a lot, mostly wondering if the word campaigner is disproportionately biasing the outputs.

**Pull request merged**